### ISSUE TITLE
Color scheme used in "What lessons..." section failing accessibility compliance checks

### ISSUE THREAD

**Issue by logan-reynolds** [deraadt/moodle-block_completion_progress#51]
It's been brought to our attention that the green and white color scheme used within the "What lessons have I viewed for this course?" section do not meet color contrast requirements to meet accessibility standards.  The green color of the background (#73A839) has a 2.8:1 contrast ratio with the white checkmark. Users with low vision might have a difficult time determining what has been completed, so a darker shade of green should be used to meet the 3:1 minimum contrast ratio required for non-text elements.
![Lessons-completed-section](https://user-images.githubusercontent.com/4237443/77099079-5a31e980-69e1-11ea-89d0-911df43615c3.png)
